documentation/projects/proposals/trust_and_safety/content_report_moderation/20230517-implementation_plan_mature_copy_change.md

```diff
@@ -0,0 +1,235 @@
+# 2023-05-17 Implementation Plan: Copy updates `mature` -> `sensitive`
+
+**Author**: @aetherunbound
+
+<!-- See the implementation plan guide for more information: https://github.com/WordPress/openverse/tree/19791f51c063d0979112f4b9f4eeace04c8cf5ff/docs/projects#implementation-plans-status-in-rfc -->
+<!-- This template is exhaustive and may include sections which aren't relevant to your project. Feel free to remove any sections which would not be useful to have. -->
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @krysal
+- [ ] @dhruvkb
+
+## Project links
+
+<!-- Enumerate any references to other documents/pages, including milestones and other plans -->
+
+- [Project Thread](https://github.com/WordPress/openverse/issues/383)
+- [Project Proposal](https://docs.openverse.org/projects/proposals/trust_and_safety/content_report_moderation/20230411-project_proposal_content_report_moderation.html)
+
+## Overview
+
+<!-- An overview of the implementation plan, if necessary. Save any specific steps for the section(s) below. -->
+
+This implementation plan concerns the copy changes necessary for updating
+references of `mature` content to `sensitive` content. These changes will be
+made to the frontend and Django API, both in code and in web-facing copy. There
+should be no design changes or new features as part of this plan, only changes
+to copy/text itself. Field and model names for the API should also be updated
+when possible, but without the need for database migrations (i.e., using
+[Django's built-in tools for specifying different underlying names for fields and tables](https://docs.djangoproject.com/en/4.2/ref/models/options/#table-names)).
+
+## Expected Outcomes
+
+<!-- List any succinct expected products from this implementation plan. -->
+
+All references to `mature` in code and copy will be updated to `sensitive`,
+except where `mature` is
+[part of a data model](https://github.com/WordPress/openverse/blob/0f90f84d4bf856297356a918e4cc4fe42d07ffc2/ingestion_server/ingestion_server/elasticsearch_models.py#L118)
+or makes the most sense to keep (e.g. when interacting with certain provider
+aspects).
+
+## Outlined Steps
+
+<!-- Describe the implementation step necessary for completion. -->
+
+The work is broken up into two services (API and frontend) and those services
+are broken into two aspects (code and copy).
+
+### API
+
+#### Code
+
+The following models will need to be updated. Since we want to avoid a migration
+with this work, they will need to
+[explicitly reference the old table names using `Meta::db_table`](https://docs.djangoproject.com/en/4.2/ref/models/options/#table-names).
```

Comment:
I'll also support a version of this IP with migrations. No need to do it now but it would be a plus to keep the DB model updated too.

Reply:
It would probably be better to do sooner than later, if it's decided that that is desirable. It'll only get more tedious as the tables grow in size, right? Specifically with respect to the fact that we'd need to keep two tables for the rename to work with zero-downtime and copying the data will take longer as the tables grow. They're all pretty small (relatively speaking) at the moment, however.

Reply:
Agreed, if we were going to change them now would be the time. I can rework this a bit to include instructions for how to approach that.